src/views/Home.vue

```diff
@@ -21,6 +21,8 @@
       :key="semester.number"
       v-model:modules="semester.modules"
       class="bg-gray-200 rounded p-2 group/semester w-64 min-w-64"
+      :title="startSemester === undefined
+        ? `${semester.number}` : startSemester.addSemesters(semester.number - 1).toString()"
```

Comment:
```suggestion
      :title="startSemester ? startSemester.addSemesters(semester.number - 1).toString() : `${semester.number}`"
```

Reply:
And is "`${}`" and ".toString()" really needed here?

Reply:
Yes